tests/test_pass_fds.py

```diff
@@ -0,0 +1,96 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2018
+# Author: Xu Han <xuhan@redhat.com>
+
+import time
+import unittest
+
+from aexpect import client
+
+
+SLEEP = 1
+
+DEVNULL = "/dev/null"
```

Comment:
more portable way is to use `os.devnull` but as the next command is posix only, we can hardcode it (perhaps it'd make sense to add `@skipUnless(os.path.exists('/proc/1/fd'), 'This test only works on posix-like systems')`.

Reply:
ok, let me check it.